### DISCUSSION TITLE
The system does not accept photo of the document

### DISCUSSION THREAD

**Discussion by XPOHUK** [community/community#108523] [GitHub Education] [answered]
### Hi everyone,

The university email address was recognized, but I was asked for additional confirmation in the form of a document. The student ID is not recognized by the system. What should I do?
My student ID:
![photo_2024-02-16_18-23-14](https://github.com/community/community/assets/39873189/f5f6b13b-0a46-4c28-9397-a3689f34944f)

**Comment by Ariffazeel99** [comment]
Submit an English translation. May be that can help

**Reply by XPOHUK** [reply]
Overlay a translation on top of an image? It will be something terrible...

**Comment by Randil-Hasanga** [comment]
I was also encountered this issue. In my case it is MS Azure. Then I contacted admin of the university. She fixed it right away. The problem is my details are not provided to Azure student DB. Try to contact your university admin.

**Comment by LukasAndreano** [comment]
К сожалению, как я понял, вряд ли что-то получится сделать)
До поддержки не достучишься, поэтому без бонусов сидеть)

**Reply by XPOHUK** [reply]
Печально. А напрямую, в связи с санкциями, JetBrains учебные лицензии не выдаёт... Буду кодить в Vim.))

**Reply by LukasAndreano** [reply]
JetBrains не выдает? А как давно? Еще в августе успешно продлял лицензии
А с GitHub - издевательство. Поддержка полностью закрыта, а нейронки специально не распознают)

**Reply by XPOHUK** [reply]
Не знаю, на сколько давно. Но вот на днях при попытке получить учебную получил ответ:
"К сожалению, мы не можем обработать ваш запрос. JetBrains не продает свои продукты и не предоставляет услуги на территориях, на которые наложены санкции ЕС, ООН или США или для которых действуют комплексные меры по контролю экспорта, поскольку это связано с нарушениями законодательства и другими правовыми и бизнес-рисками."

**Reply by XPOHUK** [reply]
Может быть попробовать ISIC (International Student Identity Card) оформить?

**Comment by XPOHUK** [comment, marked as answer]
В общем, получил ISIC. Через него получил GitHub Student Developer Pack. Через него попробовал получить JetBrains, но увы. Возможно надо было попробовать указать страну не Россия и почтовый адрес не в домене .ru. Потому что получилось напрямую через ISIC на сайте JetBrains при указании почтового адреса в домене .com и не России.

**Comment by nightcomet919** [comment]
Shoot so that all edges can be show to others, and try again.

**Comment by Mika315** [comment]
Hi  my student card  is not acceptable 
![compressed_image](https://github.com/user-attachments/assets/8c79d4cf-d7b5-4d23-bdcf-ce6abb667126)
This is the only document i have in English
